experimental/instrumentation-fetch/package.json

```diff
@@ -48,7 +48,7 @@
   },
   "homepage": "https://github.com/grafana/faro-web-sdk",
   "dependencies": {
-    "@grafana/faro-core": "^1.4.2"
+    "@grafana/faro-core": "file:.yalc/@grafana/faro-core"
```

Comment:
Why do we need yalc? 
Lerna takes care of the linking.

Does Yarc work well with lerna?